content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md

```diff
@@ -0,0 +1,429 @@
+
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Advanced/ConfigurationAPI.php";
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Advanced/CustomObjectAPI.php";
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Advanced/CustomRegistrationDataAPI.php";
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Advanced/MultiFactorAuthenticationAPI.php";
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Advanced/CustomRegistrationDataAPI.php";
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Social/NativeSocialAPI.php";
+require_once "./src/LoginRadiusSDK/CustomerRegistration/Social/SocialAPI.php";
+
+
+?>
+```
+
+Let's not forget to replace with our site API key and our API Secret.
+
+Now, let's login to your LoginRadius account and make a few changes before proceeding further.
+
+### Registering A User On LoginRadius Platform
+
+This is important to implement LoginRadius passowrdless login. To do this, you'll create:
+
+- `register.php` (contains basic html code for registering) 
+- `verify.php` 
+
+For `register.php`, paste the following code:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <title>Registration Page</title>
+      <link
+         href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+         rel="stylesheet"
+         />
+   </head>
+   <body>
+      <div class="d-flex min-vh-100 justify-content-center align-items-center">
+         <div class="shadow-sm mx-2 border rounded p-5">
+            <h3>Welcome</h3>
+            <form action="verify.php" method="POST">
+               <div class="form-group">
+                  <label  class="mt-3 mb-1 text-muted">
+                  Firstname
+                  </label>
+                  <input  type="text" name="firstname" class="form-control" />
+                  <label  class="mt-3 mb-1 text-muted">
+                  Lastname
+                  </label>
+                  <input  type="text" name="lastname" class="form-control" />
+                  <label  class="mt-3 mb-1 text-muted">
+                  Email
+                  </label>
+                  <input  type="email" name="email" class="form-control" />
+                  <label  class="mt-3 mb-1 text-muted">
+                  Password
+                  </label>
+                  <input  type="text" name="password" class="form-control" />
+                  <label  class="mt-3 mb-1 text-muted">
+                  <label  class="mt-3 mb-1 text-muted">
+                  Phone
+                  </label>
+                  <input type="text" name="phone" class="form-control" />
+                  <button name="submit" type="submit" class="btn btn-success mt-2 w-100" >
+                  Continue
+                  </button>  
+               </div>
+            </form>
+         </div>
+      </div>
+   </body>
+</html>
+```
+
+For `verify.php`, add the code as follows:
```

Comment:
@oyedeletemitope

Please add some description as to what the code programmatically does. And please feel free to add inline comments in the code. Please do the same for the remaining .PHP files wherever you have mentioned and added the code.